backend/ai_agents/data_whisperer.py

```
@@ -105,38 +105,46 @@ def analyze_data(symbol: str = "AAPL") -> Dict:
         print(f"[Data Whisperer] Real-time price for {symbol}: ${price}")
 
     # Enhanced market analysis
+    # Calculate indicators first
+    ma_20 = round(price * random.uniform(0.95, 1.05), 2)
```

Ruff (0.15.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)